[OOE-98] Created UDF to send new events to Azure Boards

ccaspanello opened the pull request.

ccaspanello committed:
[OOE-98] Created UDF to send new events to Azure Boards

DFieldFL commented on src/main/java/com/overops/azure/boards/AzureBoardsFunction.java:
```diff
@@ -0,0 +1,134 @@
+            RequestBody body = RequestBody.create(mediaType, bodyContent);
+
+            // Send Request
+            URL url = new URL("https://dev.azure.com/" + input.organization + "/" + input.project + "/_apis/wit/workitems/$task?api-version=5.0");
```
Can this hostname change?

DFieldFL replied:
SaaS is always using the same hostname.

DFieldFL commented on src/main/java/com/overops/azure/boards/view/DescriptionGenerator.java:
```diff
@@ -0,0 +1,48 @@
+     */
+    public String generate(EventView event) {
+        try {
+            TemplateLoader loader = new ClassPathTemplateLoader(){
+                @Override
+                protected URL getResource(String location) {
+                    return DescriptionGenerator.class.getResource(location);
+                }
+            };
```
Maybe add a comment on why you had to do it this way knowing there was class loading issues.

ccaspanello replied:
Good call!

ccaspanello committed:
[OOE-98]

Added comment about why we override the getResource method in the EventGenerator
